### PR TITLE
Correct typo

### PR DESCRIPTION
**Pull request by Abdur-rahmaanJ**
Subject: <short purpose of this pull request>

### Feature or Bugfix

- Bugfix

### Purpose
Correct typo.

### CODE REVIEW

**Comment by jfbu** [comment]
Thanks, your patch was on top of master branch but the PR was asking to merge on 5.x.  I force-pushed to your branch after rebasing your commit on top of current 5.x HEAD.

**Review by jfbu — COMMENTED** [review]
This LGTM but I am not native speaker so will wait for confirmation.

**Comment by AA-Turner** [comment]
Definite article seems fine here to me, I'd tend to close this PR. No real strong opinion, but I wouldn't call it a typo, more a style thing.

A

**Comment by Abdur-rahmaanJ** [comment]
@AA-Turner Removed the whole line as this fact is already mentioned earlier in the same rst file.

File: doc/usage/extensions/napoleon.rst [hunk 1]
```diff
@@ -182,7 +182,6 @@ tends to use more horizontal space. Google style tends to be easier to
 read for short and simple docstrings, whereas NumPy style tends be easier
 to read for long and in-depth docstrings.
 
-The `Khan Academy`_ recommends using Google style.
```
Comment (jfbu) [inline doc/usage/extensions/napoleon.rst]:
```suggestion
```
(i.e. remove either now empty line 184 or 185)

**Review by jfbu — COMMENTED** [review]
LGTM with nits

**Comment by jfbu** [comment]
Thanks, merged! I squashed to remove "typo" mention from commit message.

**Pull request merged**